.travis.yml

```diff
@@ -2,16 +2,14 @@ sudo: false
 language: python
 
 python:
-    - 2.7
-    - 3.4
-    - 3.5
```

Comment:
nit: why are we dropping python 3.4 and 3.5?

Reply:
If you read the current (WIP) commit message above you note that these versions have reached their EOL.